src/api/renderer.c

```diff
@@ -275,22 +278,27 @@ static int f_show_debug(lua_State *L) {
 
 
 static int f_get_size(lua_State *L) {
-  int w, h;
-  ren_get_size(window_renderer, &w, &h);
+  int w = 0, h = 0;
+  if (active_window_renderer)
```

Comment:
slightly curious, how will this behave when `get_size()` is called outside of begin/end_frame()? Is even possible (for example inside a thread)?

Reply:
since `ren_get_size` isn't called it will simply return 0,0 for compatibility sake.

This will allow us to keep all existing code working (even if it shouldn't) and not behave incorrectly.